src/components/phase/Phase.test.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Map } from 'immutable';
+import SplitPane from 'react-split-pane';
+import { Phase } from './Phase';
+import SpaceDescription from '../space/SpaceDescription';
+import PhaseDescription from './PhaseDescription';
+import PhaseItems from './PhaseItems';
+import ToolsButton from '../space/ToolsButton';
+import { MAX_TOOLS_WIDTH, MIN_TOOLS_WIDTH } from '../../config/layout';
+
+const samplePhase = Map({
+  id: 'phaseId',
+  description: 'phaseDescription',
+  items: [{ id: 'phaseItemId' }],
+});
+
+const sampleTools = [{ id: 'spaceItempsId' }];
```

Comment:
Typo `Itemps`. Did you mean `Items`?

Reply:
Oops, you are right I'll correct it and merge the PR.